### ISSUE TITLE
Achievements notices

### ISSUE THREAD

**Issue by EPuncker** [Znote/ZnoteAAC#390]
When enabling the achievements in config.php it throws some notices:

```
Notice: Undefined index: secret in C:\UniServerZ\www\achievements.php on line 44

Notice: Undefined index: img in C:\UniServerZ\www\achievements.php on line 35

Notice: Undefined index: img in C:\UniServerZ\www\achievements.php on line 31

Notice: Undefined index: secret in C:\UniServerZ\www\achievements.php on line 44

Notice: Undefined index: img in C:\UniServerZ\www\achievements.php on line 27

Notice: Undefined index: secret in C:\UniServerZ\www\achievements.php on line 44

Notice: Undefined index: img in C:\UniServerZ\www\achievements.php on line 24
```

seems like it doesn't have a default value for img and secret

**Comment by Znote** [comment]
Is this fixed in #391 ?

**Comment by EPuncker** [comment]
> Is this fixed in #391 ?

no it is not, we (you 😄, since I don't know php) need to add a default value in the code to be used when these are undefined
